src/redux/walletconnect.js

```diff
@@ -109,33 +109,31 @@ export const walletConnectOnSessionRequest = (
         const { peerId, peerMeta } = payload.params[0];
         const imageUrl = get(peerMeta, 'icons[0]');
         const name = convertDappNameToDisplay(peerMeta.name);
-        InteractionManager.runAfterInteractions(() => {
```

Comment:
I think this was here for a reason

Reply:
If you remove this you won't see the animation of the WC prompt sheet sliding from the bottom while doing mobile deeplinking.

Reply:
@brunobar79 i see the animation just fine! i think this change is actually ok. also, this particular line was like the main culprit for why i was having problems scanning QR Codes sometimes

Reply:
I tested it and that’s what I saw but if you say it works then great.